Assets/DragDrop.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public enum DropType
+{ 
+    None,
+    Head,
+    Hand,
+    Foot,
+    Heart
+}
+
+
+public class DragDrop : MonoBehaviour, IPointerDownHandler, IBeginDragHandler, IEndDragHandler, IDragHandler
+{
+    public RectTransform rectTransform;
+    public DropType dropType = DropType.None;
+
+    private Transform lastPosition;
+    private Canvas canvas;
+    private CanvasGroup canvasGroup;
+
+    void Start()
+    {
+        canvas = PlanningSceneManager.Instance.gameCanvas;
+        rectTransform = GetComponent<RectTransform>();
+        canvasGroup = GetComponent<CanvasGroup>();
+    }
+
+    public void OnPointerDown(PointerEventData eventData)
+    {
+        //Not Implemented
```

Comment:
nit pick but you could do `throw new NotImplementedException();` or `Debug.Log("OnPointerDown ran " + eventDate)`

Reply:
Good Point, if I'm not using it I should just remove the interface